src/routes/+page.svelte

```diff
@@ -71,6 +73,9 @@
 				break;
 			}
 		}
+		const md = markdownit('commonmark');
+		history[history.length - 1].content = md.render(history[history.length - 1].content);
```

Comment:
Similar here we may want to put this in the backend, maybe by making a custom output parser in langchain to sanitize the output and render the markdown at html

Reply:
How would I get edit the readable stream? this.model.pipe(new BytesOutputParser()).stream(chat_history);